v3/src/lib/cfm/use-cloud-file-manager.ts

```diff
@@ -182,6 +182,79 @@ function getHelpUrl() {
   return translatedHelpURLs[gLocale.current] || helpURL
 }
 
+function getMenuBar(cfm: CloudFileManager) {
+  return {
+    onInfoClick() {
+      window.open(projectWebSiteURL, "_blank")
+    },
+    subMenuExpandIcon: SubMenuExpandIcon,
+    otherMenus: [
+      {
+        className: "help-menu",
+        menuAnchorIcon: HelpIcon,
+        menuAnchorName: t("DG.ToolButtonData.help.title"),
+        menu: [
+          {
+            icon: HelpPagesIcon,
+            name: t("DG.AppController.optionMenuItems.help"),
+            action: () => window.open(getHelpUrl(), "_blank")
+          },
+          {
+            icon: HelpForumIcon,
+            name: t("DG.AppController.optionMenuItems.help-forum"),
+            action: () => window.open(helpForumURL, "_blank")
+          },
+          {
+            icon: CODAPProjectIcon,
+            name: t("DG.AppController.optionMenuItems.toWebSite"),
+            action: () => window.open(projectWebSiteURL, "_blank")
+          },
+          {
+            icon: PrivacyPolicyIcon,
+            name: t("DG.AppController.optionMenuItems.toPrivacyPage"),
+            action: () => window.open(privacyPolicyURL, "_blank")
+          }
+        ]
+      },
+      {
+        className: "settings-menu",
+        menuAnchorIcon: SettingsIcon,
+        menuAnchorName: t("DG.ToolButtonData.optionMenu.title"),
+        menu: [
+          {
+            icon: ToolbarPositionIcon,
+            name: `${t("V3.AppController.optionMenuItems.positionToolShelf")} ${persistentState.toolbarPosition}`,
```

Comment:
Should use either variable replacement or two separate strings here rather than string concatenation because the order may be different in other languages. In this case I think two separate strings is simpler:
```suggestion
            name: t(`V3.AppController.optionMenuItems.toolbarPosition${persistentState.toolbarPosition}`,
```